src/config/site.config.jsx

```diff
@@ -21,9 +21,6 @@ module.exports = {
     /* The title the site header should display. */
     title: '', // maps to appTitle
 
-    /* The subtitle the site header should display. */
-    subtitle: '', // maps to appSubtitle
-
     /** The themes to supply the ThemeProvider which allows the site to switch
```

Comment:
This file doesn't look like it has any JSX in it, why the change from `.js` to `.jsx`?

Reply:
It allows us to import non-compiled files, so 

before I was dynamically importing the [`geneartedComponentConfig file`](https://github.com/cerner/terra-site/pull/15/files#diff-a7b1a86dc64b7d308e86bbc8f33e449aL26) in `Index.jsx`. This worked fine, however when I started working on terra-ui.com I realized I had removed the ability to add custom home, page component and menu components in the navigation.config. For example, see [here](https://github.com/cerner/terra-ui.com/compare/add-core-and-framework#diff-bc22fb7ebfd46781a950719ac185b30dR18).


With this chance we can directly pass the generatedComponentConfig and navigation config as objects and we reduce the need to [check for multiple file config and/or provide a defaults](https://github.com/cerner/terra-site/pull/15/files#diff-ed7a1de5bf46235874aea14f774f5b51L26) otherwise and it allows us to reduce the number of [globally defined variables from 2 to 1](https://github.com/cerner/terra-site/pull/15/files#diff-ed7a1de5bf46235874aea14f774f5b51L106).

Reply:
😑  well... after writing this, then seeing your comment above I went and changed some things in terra-ui. I am going to investigate a bit further because it seems this might not actually need to be changed.